app/design/frontend/Magento/blank/Magento_Checkout/web/css/source/module/checkout/_checkout-agreements.less

```diff
@@ -8,6 +8,17 @@
 //  _____________________________________________
 
 & when (@media-common = true) {
+    @color_1: #e02b27;
+    .checkout-agreement.field.choice.required {
+        display: flex;
+        &::after {
+            color: @color_1;
+            content: '*';
+            font-size: 1.2rem;  
+            margin: 3px 0 0 5px;
```

Comment:
I'm curious why this was needed! `.required` should already be covered by `lib-form`'s output. Does [`.required > .label`](https://github.com/magento/magento2/blob/d93d28893692a9fce2088ff80abd25f411219fbb/lib/web/css/source/lib/_forms.less#L817) not match here?

If it _is_ needed, should probably use variables here so that changing  `@form-field-label-asterisk` variables will affect it.

Something like:

https://github.com/magento/magento2/blob/d93d28893692a9fce2088ff80abd25f411219fbb/app/design/frontend/Magento/blank/Magento_Catalog/web/css/source/_module.less#L257-L265

Reply:
Hi
As the issue is because earlier Magento put "term and condition" in label element and after that attached asterisk via CSS(.label) so the asterisk move according to the size of the text of "term and condition."

what I did remove the label class and put a new CSS class for asterisk around the div that contains the "term and condition label"